extensions/blocks/story/editor.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../shared/styles/gutenberg-base-styles.scss';
+@import './player/style.scss';
+
+.wp-block-jetpack-story__add-item {
+	margin-top: 4px;
+	width: 100%;
+
+	.components-form-file-upload,
+	.components-button.wp-block-jetpack-story__add-item-button {
+		width: 100%;
+		height: 100%;
+	}
+
+	.components-button.wp-block-jetpack-story__add-item-button {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		box-shadow: none;
+		border: none;
+		border-radius: 0;
+		min-height: 100px;
+
+		.dashicon {
+			margin-top: 10px;
+		}
+
+		&:hover,
+		&:focus {
+			border: 1px solid $dark-gray-500;
+		}
+	}
+}
+
+.wp-story-container {
+
+	.wp-story-prev-slide,
+	.wp-story-next-slide {
+		button {
+			width: 36px !important;
+			height: 36px !important;
+			color: #555d66;
+			border: 1px solid #555d66;
+			outline: 0;
+			background-color: transparent;
+
+			&:hover {
+				border: 1px solid #555d66;
+				background-color: transparent;
+				i {
+					color: rgb(51, 129, 184);
```

Comment:
Can we rely on Core colors for these elements as well?

Reply:
Seems like this is the same styles defined by `components-button`. I still need to figure out how to have that class on the block editor and not on the frontend side

Reply:
Ok I would need to make use of the Button component in core and add some customized styles to make it round and so on. That would mean replacing the font with SVG (which is totally fine), removing my custom button components and updating the various instances. If we want to make it for the code freeze today I suggest I just add that to the tasks for next iteration if that's fine with you

Reply:
That works for me 👍